unstructured_ingest/v2/processes/connectors/qdrant.py

```diff
@@ -0,0 +1,223 @@
+import json
+import multiprocessing as mp
+from typing import TYPE_CHECKING,Optional,List,Dict,Any
+import uuid
+from dataclasses import dataclass, field
+
+from pathlib import Path
+
+from datetime import date, datetime
+
+from dateutil import parser
+from pydantic import Field
+
+from unstructured_ingest.enhanced_dataclass import enhanced_field
+from unstructured_ingest.error import DestinationConnectionError, WriteError
+from unstructured_ingest.v2.interfaces import (
+    AccessConfig,
+    ConnectionConfig,
+    FileData,
+    Uploader,
+    UploaderConfig,
+    UploadStager,
+    UploadStagerConfig,
+)
+from unstructured_ingest.v2.logger import logger
+from unstructured_ingest.utils.data_prep import batch_generator, flatten_dict
+from unstructured_ingest.utils.dep_check import requires_dependencies
+from unstructured_ingest.v2.processes.connector_registry import DestinationRegistryEntry
+
+if TYPE_CHECKING:
+    from qdrant_client import QdrantClient
+
+
+CONNECTOR_TYPE = "qdrant"
+
+
+class QdrantAccessConfig(AccessConfig):
+    api_key: Optional[str] = Field(default=None, sensitive=True, description="API Key")
+
+
+
+class QdrantConnectionConfig(ConnectionConfig):
+    collection_name: str
```

Comment:
These should all have a `Field(description=...` since the CLI text help is based on that. Copy it from the previous CLI config.